docs/guides/working-with-json/json-structured.md

```diff
@@ -0,0 +1,84 @@
+---
+sidebar_label: Relying on Structure 
+sidebar_position: 2
+description: Using a structured approach
+---
+
+# Structured Approach
+
+First, we confirm we can read the JSON dataset and highlight the challenges of handling semi-structured data using more traditional types used in other databases. We don’t rely on Schema inference to map the JSON fields to columns in the example below - instead, we specify a format of JSONEachRow and map the fields explicitly to columns in the s3 functions. 
+
+```sql
+SELECT type, `actor.display_login`, `repo.name`, created_at
```

Comment:
I'm getting an access error:

Received exception from server (version 22.2.2):Code: 499. DB::Exception: Received from localhost:9000. DB::Exception: Access Denied. (S3_ERROR)

Reply:
resolved

Reply:
it works now